templates/SilverStripe/ElementalBannerBlock/Block/BannerBlock.ss

```diff
@@ -1,27 +1,32 @@
-<%-- Display the image (File) --%>
-<% if $File %>
-    <span class="banner-element__image">
-        $File
-    </span>
-
-<% end_if %>
-<div class="banner-element__content">
-    <% if $Title && $ShowTitle %>
-        <h2 class="banner-element__title">$Title</h2>
-    <% end_if %>
+<a class="banner-block"
+  <% if $CallToActionLink.Page.Link %>
+  <% with $CallToActionLink %>
+  href="{$Page.Link}"
+  <% if $TargetBlank %>target="_blank"<% end_if %>
+  <% if $Description %>title="{$Description.ATT}"<% end_if %>
+  <% end_with %>
+  <% end_if %>
+>
+<span class="banner-block__flex-box" role="banner">
+    <span class="banner-block__content">
+        <% if $Title && $ShowTitle %>
+            <span role="heading" class="banner-block__title h4">$Title</span>
+        <% end_if %>
+        <p>$Content.Summary(30)</p>
+        <% if $CallToActionLink.Text %>
+        <span class="banner-block__call-to-action">
+            {$CallToActionLink.Text.XML}
+            <img class="banner-block__call-to-action-icon" src="$resourceURL('themes/bambusa/images/icon-arrow.svg')" alt="" />
```

Comment:
We'll probably get dinged by this in accessibility checkers - can you see if there's a specific ARIA role for "ignore this image"?

Reply:
An empty `alt` attribute is the appropriate way to specify that an image is only use for a presentational purpose. Accessibility tools will usually flag the absence of an `alt` attribute as a problem. By providing an empty `alt`, we're in effect stating that we've considered whether this image should have an alternative text and concluded that it shouldn't.

I've tested this with the AXE extension and the image didn't get flag.

https://webaim.org/techniques/alttext/

Reply:
Nice, I did not know that!